src/basic/user-util.h

```diff
@@ -60,13 +60,6 @@ int take_etc_passwd_lock(const char *root);
 #define UID_INVALID ((uid_t) -1)
 #define GID_INVALID ((gid_t) -1)
 
-/* Let's pick a UIDs within the 16bit range, so that we are compatible with containers using 16bit
- * user namespacing. At least on Fedora normal users are allocated until UID 60000, hence do not
- * allocate from below this. Also stay away from the upper end of the range as that is often used
- * for overflow/nobody users. */
```

Comment:
Maybe move this comment to `meson_options.txt`?

Reply:
Well, the reason I put the UID-GID.md text together, was to move it there, and it's in there. I figure that#s the best place for explaining this

Reply:
OK.